ci/cloudbuild.yaml

```diff
@@ -1,84 +1,105 @@
 steps:
+  # This step will fetch the entire git history so tags can be accessed during
+  # the build. If this negatively impacts build performance, please modify it so
+  # only commits up to the latest tag are fetched.
+  - name: "gcr.io/cloud-builders/git"
+    id: "Fetch tags from git"
+    args: [fetch, --unshallow]
 
-# This step will fetch the entire git history so tags can be accessed during
-# the build. If this negatively impacts build performance, please modify it so
-# only commits up to the latest tag are fetched.
-- name: 'gcr.io/cloud-builders/git'
-  id: 'Fetch tags from git'
-  args: [fetch, --unshallow]
+  - name: "gcr.io/cloud-builders/git"
+    id: "Get version from git tags"
+    entrypoint: "/bin/bash"
+    args: ["-c", "git describe --abbrev=0 --tags > _release_tag"]
 
-- name: 'gcr.io/cloud-builders/git'
-  id: 'Get version from git tags'
-  entrypoint: '/bin/bash'
-  args: ['-c', 'git describe --abbrev=0 --tags > _release_tag']
+  - name: "python:3"
+    id: "Create version tags"
+    args: ["python3", "ci/create_version_tags.py"]
 
-- name: 'python:3'
-  id: 'Create version tags'
-  args: ['python3', 'ci/create_version_tags.py']
+  # TODO(https://github.com/google/ts-bridge/issues/67): This step will mask all
+  # errors from docker pull, not just when image doesn't exist. Consider using
+  # a more optimal approach which can parse the output from docker pull to handle
+  # errors better.
+  - name: "gcr.io/cloud-builders/docker"
+    entrypoint: "/bin/bash"
+    args: ["-c", "docker pull gcr.io/$PROJECT_ID/ts-bridge:latest || exit 0"]
 
-# TODO(https://github.com/google/ts-bridge/issues/67): This step will mask all
-# errors from docker pull, not just when image doesn't exist. Consider using
-# a more optimal approach which can parse the output from docker pull to handle
-# errors better.
-- name: 'gcr.io/cloud-builders/docker'
-  entrypoint: '/bin/bash'
-  args: ['-c', 'docker pull gcr.io/$PROJECT_ID/ts-bridge:latest || exit 0']
+  - name: "gcr.io/cloud-builders/docker"
+    id: "initialize-qemu"
+    args:
+      ["run", "--privileged", "multiarch/qemu-user-static", "--reset", "-p yes"]
```

Comment:
Can you specify what `-p yes` and `--reset` are needed for?

Reply:
Done :)